lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -284,6 +295,27 @@ def get_target_from_rule(module, ec2, rule, name, group, groups, vpc_id):
     return group_id, ip, target_group_created
 
 
+def ensure_tags(ec2, group, tags, changed, module):
+    try:
+        resource_id = group.id
+        cur_tags = group.tags
+        if cur_tags == tags:
```

Comment:
You're right about 'TRUE' being an issue. I don't know what to do about that. I'm lukewarm on failing since I can't think of any other cornercases. It shouldn't ever cause issues for integers or floats. Try creating a tag with the string "1" or "1.838" with your PR and if you check in the AWS console it rightly interprets them as numbers. But when boto is reading those numbers for us to compare they will always be strings...